Update humans_are_at_the_top_of_the_food_chain.md (#37)
veganhacktivists/animalsupportbot: knowledge/responses/humans_are_at_the_top_of_the_food_chain.md

> The food chains within nature are incredibly important, they symbolise something that is part of the natural order and exists within an eco-system. They help to maintain healthy populations of animals and ensure that the natural ecology is balanced. 

Reviewer: ```suggestion
The food chain within nature is incredibly important, as it symbolizes something that is part of the natural order and exists within an ecosystem. It helps to maintain healthy populations of animals and ensures that the natural ecology is balanced. 
```

> The food chains in nature are a consequence of the dynamics of the wild world. The predator needs to kill the prey to survive. Prey needs to kill to defend themselves. However, <a href="https://www.dailymail.co.uk/news/article-422784/How-leopard-changed-spots---saved-baby-baboon.html">not even predators are without mercy</a>. That is because once the <b>need</b> to kill for survive is remove from the equation, being stronger is not a justification for killing mindlessly.

Reviewer: Bring source to "References" section

> The food chains in nature are a consequence of the dynamics of the wild world. The predator needs to kill the prey to survive. Prey needs to kill to defend themselves. However, <a href="https://www.dailymail.co.uk/news/article-422784/How-leopard-changed-spots---saved-baby-baboon.html">not even predators are without mercy</a>. That is because once the <b>need</b> to kill for survive is remove from the equation, being stronger is not a justification for killing mindlessly.

Reviewer: ```suggestion
The food chain in nature is a consequence of the dynamics of the wild world. The predator needs to kill the prey to survive. Prey needs to kill to defend themselves. However, <a href="https://www.dailymail.co.uk/news/article-422784/How-leopard-changed-spots---saved-baby-baboon.html">not even predators are without mercy</a>. Once the need to kill for survive is removed from the equation, being stronger is not a justification for killing mindlessly.
```

> Flesh and other animal product are <a href="https://www.cambridge.org/core/journals/proceedings-of-the-nutrition-society/article/plantbased-diets-and-longterm-health-findings-from-the-epicoxford-study/771ED5439481A68AD92BF40E8B1EF7E6"><b>not</b> needed for nutrition</a>. Humankind consumes it purely for sensory pleasure. Therefore, we don’t have to kill anyone to live. Our position as the species on the top of the food chain doesn't have to use our dominance to kill others. 

Reviewer: Bring citation to a "References" section. Since it's a study, you should use APA citation:
```
Key, T., Papier, K., & Tong, T. (2021). Plant-based diets and long-term health: Findings from the EPIC-Oxford study. Proceedings of the Nutrition Society, 1-9. doi:10.1017/S0029665121003748
```

> Flesh and other animal product are <a href="https://www.cambridge.org/core/journals/proceedings-of-the-nutrition-society/article/plantbased-diets-and-longterm-health-findings-from-the-epicoxford-study/771ED5439481A68AD92BF40E8B1EF7E6"><b>not</b> needed for nutrition</a>. Humankind consumes it purely for sensory pleasure. Therefore, we don’t have to kill anyone to live. Our position as the species on the top of the food chain doesn't have to use our dominance to kill others. 

Reviewer: ```suggestion
Flesh and other animal products are not needed for nutrition [2]. Humankind consumes it purely for sensory pleasure. Therefore, we don’t have to kill anyone to live. Our position as the species on top of the food chain does not necessitate the use of our dominance to kill others. 
```

> Not only we are the most fit - we are rghe only ones capable of being moral agents. This means we can weigh the consequences and impact of our actions.  We can make decisions based on a notion of right and wrong and most importantly we can be held accountable for the actions that we make. So, as the dominant species, we can use our power to look after others and create a better world - in fact, we have a moral obligation to. Our moral agency dictates to us that when we are in a situation where we have the choice to not inflict unnecessary suffering we can and should be held accountable if we actively choose to inflict unnecessary suffering instead.

Reviewer: ```suggestion
Not only we are the most —we are the only ones capable of being moral agents. This means we can weigh the consequences and impact of our actions.  We can make decisions based on the notion of right and wrong and most importantly, we can be held accountable for the actions that we make. As the dominant species, we can use our power to look after others and create a better world—in fact, we have a moral obligation to do so. Our moral agency dictates that if we are given the choice whether to inflict unnecessary suffering, choosing to actively do so means that we can and should be held accountable.
```

> Finally, what we do to is far removed from the natural world. We selectively breed, artificially inseminate, mutilate and farm them, before driving them in trucks to the slaughterhouse where we hang them upside down, kill them and drain them of their blood is about as far removed from nature as possible and does not resemble anything like a food chain. 

Reviewer: ```suggestion
Finally, what we do to animals is far removed from the natural world. We selectively breed, artificially inseminate, mutilate and farm them, before driving them in trucks to the slaughterhouse where we hang them upside down and kill them is about as far removed from nature as possible. This does not resemble anything like a food chain. 
```

Reviewer: I would also add a source for this.
